### ISSUE TITLE
ARM64: COM Exceptions when running tests

### ISSUE THREAD

**Issue by ianhays** [dotnet/corefx#10921]
I'm getting a ton of COM Exceptions when running the ARM64 tests.

Ex:

```
System.Xml.Tests.DocumentElement_GetElementsByTagNameTests.GetCaseSensitiveElements()
15:46:50    System.Xml.Tests.NodeInsertingTests.CreateElementAndModifyInnerText [FAIL]
15:46:50       System.Runtime.InteropServices.COMException : Unspecified error (Exception from HRESULT: 0x80004005 (E_FAIL))
15:46:50       Stack Trace:
15:46:50             at System.Delegate.GetMulticastInvoke()
15:46:50             at System.MulticastDelegate.NewMulticastDelegate(Object[] invocationList, Int32 invocationCount, Boolean thisIsMultiCastAlready)
15:46:50             at System.MulticastDelegate.CombineImpl(Delegate follow)
15:46:50             at System.Xml.Tests.NodeInsertingTests.CreateElementAndModifyInnerText()
```

Logs: http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/master/job/arm64_windows_nt_release/26/consoleFull

@kyulee1

**Comment by kyulee1** [comment]
I think we should do a binary search to find the offending merge -- either coreclr or corefx.
I also noticed that the current automation never succeeds since there are expected failures. We should mark them skipped so that we can catch any regression like this automatically.

**Comment by ianhays** [comment]
The first instance of this failure occurred in the [August 11th build](http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/master/job/arm64_windows_nt_release/20/). August 10th passed without issue. So we should look for changes on that date.

>We should mark them skipped so that we can catch any regression like this automatically.
There isn't a good way of doing this with the CoreFX tests as far as I'm aware.

**Comment by kyulee1** [comment]
I think the offending merge is https://github.com/dotnet/coreclr/commit/3961c702b9b9cce2a2bab1792446f40b19ddb260

**Comment by rahku** [comment]
Fixed in dotnet/coreclr@473d6f1e7e30b56aaa7677da45163bc567210a62
